javascript/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 0.0.6
 
+* Moved `jobScheduler` to [ha-job-scheduler](https://www.npmjs.com/package/ha-job-scheduler).
```

Comment:
Should we do a minor version change on this one at least since it s a pretty significant change?

Reply:
Great idea!